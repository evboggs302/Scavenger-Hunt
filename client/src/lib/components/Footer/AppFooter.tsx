import React from "react";
import { License } from "./License";

export const AppFooter = () => {
  return <License />;
};
