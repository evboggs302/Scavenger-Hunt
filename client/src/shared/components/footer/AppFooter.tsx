import React from "react";
import { Layout } from "antd";
import { Copyright } from "../Copyright";

const { Footer } = Layout;

export const AppFooter = () => {
  return (
    <Footer>
      <Copyright />
    </Footer>
  );
};
