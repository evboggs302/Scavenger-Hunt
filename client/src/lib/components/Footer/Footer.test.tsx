import { render } from "@testing-library/react";
import { AppFooter } from "./AppFooter";

describe("Footer", () => {
  it("Should match snapshot", async () => {
    const component = render(<AppFooter />);
    expect(component).toMatchSnapshot();
  });
});
