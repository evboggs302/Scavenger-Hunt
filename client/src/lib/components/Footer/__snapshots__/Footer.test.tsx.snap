// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer > Should match snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      MIT License 
      <a
        color="inherit"
        href="https://github.com/evboggs302/Scavenger-Hunt/blob/master/LICENSE"
        rel="noreferrer"
        target="_blank"
      >
        "Digital Scavenger" by Evan Boggs
      </a>
       
      2025
      .
    </div>
  </body>,
  "container": <div>
    MIT License 
    <a
      color="inherit"
      href="https://github.com/evboggs302/Scavenger-Hunt/blob/master/LICENSE"
      rel="noreferrer"
      target="_blank"
    >
      "Digital Scavenger" by Evan Boggs
    </a>
     
    2025
    .
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
